pkg/controller/machine_safety.go

```diff
@@ -358,7 +382,8 @@ func (c *controller) deleteOrphanVM(vm driver.VM, secretRef *corev1.Secret, kind
 	err := dvr.Delete()
 	if err != nil {
 		glog.Warning("Error while deleting VM on CP - ", err)
```

Comment:
Can we have it as Error and not Warning. Also please describe in the logs that, controller will retry after resync to delete it again.